l10n_br_account_payment_order/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today - KMEE (<http://kmee.com.br>).
+#  Luis Felipe Miléo - mileo@kmee.com.br
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Brazilian Payment Order',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': "KMEE, "
+              "Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/l10n-brazil',
+    'category': 'Banking addons',
+    'depends': [
+        'l10n_br',
```

Comment:
This depedency is not necessary because a Company can create own chart of account